src/Roslyn.Suppressions/Roslyn.Suppressions/packages.lock.json

```diff
@@ -116,14 +116,6 @@
         "resolved": "2.2.85",
         "contentHash": "YptRsDCQK35K5FhmZ0LojW4t8I6DpetLfK5KG8PVY2f6h7/gdyr8f4++xdSEK/xS6XX7/GPvEpqszKVPksCsiQ=="
       },
-      "Microsoft.Alm.Authentication": {
```

Comment:
This change and the ones below in this file are weird. Why have all of these references been removed? They're nothing to do with the SonarAnalyzers.